website/docs/flow/stages/authenticator_validate/index.md

```diff
@@ -38,7 +38,7 @@ Requires authentik 2021.12.4
 :::
 
 :::caution
-Firefox has some known issues regarding FIDO (see https://bugzilla.mozilla.org/show_bug.cgi?id=1530370) and TouchID (see https://bugzilla.mozilla.org/show_bug.cgi?id=1536482)
+Firefox has some known issues regarding TouchID (see https://bugzilla.mozilla.org/show_bug.cgi?id=1536482)
```

Comment:
I think we might be able to fully remove this, I remember reading something that TouchID works on Firefox now @gergosimonyi cc @rissson

Reply:
Offline resolution: let's keep it in, seems like it still doesn't work